### ISSUE TITLE
Show "i"-icon for tag tourism=visitor_centre (again) like for tag tourism=information

### ISSUE THREAD

**Issue by sfkeller** [gravitystorm/openstreetmap-carto#4773]
### Expected behavior

Show "i"-icon for tag tourism=visitor_centre (again) like for tag tourism=information.

### Actual behavior

Does not show "i"-icon (anymore) because of issue #4247: see comment https://github.com/gravitystorm/openstreetmap-carto/issues/4247#issuecomment-1400536517 

I like the idea of this issue. But it seems that not all consequences were taken into account when resolving issue #4247 .

There is information=visitor_centre, which is very similar to information=office but has a crucial difference, as it is correctly described in the wiki https://wiki.openstreetmap.org/wiki/Tag:information%3Dvisitor_centre .

In my opinion, information=visitor_centre must be represented with an "i"-icon as before, like information=office. 

There are currently worldwide 417 tags "information=visitor_centre" https://taginfo.openstreetmap.org/tags/information=visitor_centre , e.g. here https://www.openstreetmap.org/node/1330855837

In order to emphasize that visitor_centre is close to office see how the iD-Editor in the feature selection (when entering an object) explicitly displays "Visitor Centre" as aka broader tag of "Information Office" - and then saves the tag information=office: see e.g. here: https://www.openstreetmap.org/node/5806243475 and screenshot below.

### Screenshots with links illustrating the problem
![grafik](https://user-images.githubusercontent.com/43340/214590741-5365753d-0bdc-4b7d-95d3-ded270e64581.png)

**Comment by imagico** [comment]
Thanks.

I assume you are talking about `information=visitor_centre` (417 uses), not `tourism=visitor_centre` (13 uses).

In principle i think this is a good idea, even if the volume of use of the tag is still very low.

Looking at the tag pages on the wiki and tag use i am, however, a bit wary about the meaning of the tags.  There seems no doubt that `information=office` is not only being used for general purpose tourist information offices but also for thematically specialized places.  Hence it seems the delineation between the two tags in practical use is not very clear so far.

Related but different issue:  `information=office` is implicitly (through the term office) for _staffed_ places.  This is not clear for `information=visitor_centre`.  So it is not clear if a non-staffed room with some information leaflets and information boards and maybe a small exhibition or a self-service video presentation qualifies as a `information=visitor_centre`.

This is not meant to invite a discussion here, just to point out that there is some work to do on the tag development level.  Better clarity on the tag semantics would also allow choosing more precise and more intuitive symbols.  I would for example like to see the symbol for `information=office` to move from the current extremely abstract 'i' in circle to something indicating a staffed place.  But that would of course require using a different symbol for `information=visitor_centre` if the latter is not necessarily a staffed place.

**Comment by sfkeller** [comment]
> I assume you are talking about `information=visitor_centre` (417 uses), not tourism=visitor_centre (13 uses).

Right. Sorry.

> In principle i think this is a good idea, even if the volume of use of the tag is still very low.

I expected that you point to the current low volume of `information=visitor_centre`. I did no yet quantify my thesis, but I'd say that currently there's a very high chance that people map for the renderer and choose `information=office` instead of `information=visitor_centre`. 

Your hint that office implies staff and visitor_centre does not, is correct. But for a first step I could live with both getting the 'i' icon, even without waiting for further development of the tags. By the way, I know that in tourism the trend is to replace staffed information offices with `information=terminal` (which get's a framed 'i' icon).

So, in summary, I still like to advocate for simply adding `information=visitor_centre` also - or again - with the 'i' icon.

**Comment by map-per** [comment]
I can do the coding if you want. But I don't know how to design good icons.

**Comment by imagico** [comment]
I have designed separate symbols for `information=office` and `information=visitor_centre` along the lines of what i discussed in https://github.com/gravitystorm/openstreetmap-carto/issues/4773#issuecomment-1403842929 now for the ac-style - see https://github.com/imagico/osm-carto-alternative-colors/commit/83cb6ec225a38c086838fe17fa1352cfcab5d14e.

![tourism=information](http://www.imagico.de/files/tourism_information.png)

I like to emphasize again that i do not know if use of the tag as is warrants such a differentiation (i.e. if `information=office` is indeed consistently used for staffed places and `information=visitor_centre` for something distinctly different).  But only if such a differentiation indeed exists would it be advisable for us to actually render both tags.

**Comment by map-per** [comment]
Thanks! I like the `visitor_centre` icon. 
For the `office` icon I think that the current icon is more recognizable ('i' is usually the "standart" symbol for a tourist information).

From the random `visitor_centre` samples i looked at arround the world it appears that this tag is consistently used for information on a specific attraction.

As far as I know the `information=office` tag for tourist informations that offer broader information on the whole region or town is misused quite often. E.g. for things that are rather an `amenity=reception_desk` or for the Service Points of the Deutsche Bahn (tagging them is difficult as they don't sell tickets but just offer help for customers).

I think that the relatively new `visitor_centre` tag is a good idea and I hope that it will contribute to reducing the misuse of `office`

**Comment by imagico** [comment]
Keep in mind that this is a global map design project and what you recognize as a well established way to symbolize certain things is typically not so for the vast majority of our target map users.

> As far as I know the information=office tag for tourist informations that offer broader information on the whole region or town is misused quite often

Please no discussion here on what is and what is not correct use of certain tags.  We look at how tags are actually used and our choices in map design aim to reflect that.  We decidedly aim not to try to persuade mappers to change the way they map things to what we consider to be more correct.

**Comment by map-per** [comment]
You asked whether `visitor_centre` and `office` are used consistently, or that's at least what I understood.

The phrases "information on a specific attraction" and "information on the whole region or town" are coppied form the wiki pages

**Comment by imagico** [comment]
> The phrases "information on a specific attraction" and "information on the whole region or town" are coppied form the wiki pages

The wiki does not typically describe de facto use of tags, it describes the ought-to-be use from the subjective perspective of those who have written the wiki page.  See also https://github.com/gravitystorm/openstreetmap-carto/issues/4773#issuecomment-1403842929

**Comment by map-per** [comment]
I know

**Comment by sfkeller** [comment]
@imagico wrote:

> I have designed separate symbols

Nice. I'm happy with any icon for `information=visitor_centre` too, like `information=office`.

**Comment by map-per** [comment]
Updated the PR to include then new icon for `visitor_centre`

**Comment by imagico** [comment]
#4796 needs a review - and preferably by someone other than me since i have suggested a different design above, hence i am likely a bit prejudiced towards that.

Yes, a sample rendering would be important too.  As would be an explanation of what the choice of symbols (circled 'i' for `information=office`, 'i' within hut symbol for `information=visitor_centre`) is meant to communicate.  In https://github.com/gravitystorm/openstreetmap-carto/issues/4773#issuecomment-1493067699 i explained the idea behind the symbols i suggested and also made clear that i am not sure if this idea is supported by existing mapping practice.  I am not quite sure at the moment what idea is behind the choice of symbols in #4796.

**Comment by map-per** [comment]
I find the new `office` symbol to be rather cluttered with three different objects. And a person combined with a counter is rather ambiguous, so the 'i' in the background, at least for me, becomes the main recognizable feature.

But apart from that I think that adding `visitor_centre` and changing the `office` icon are two different things. So I wouldn't address them in one PR. Especially I don't want to make a PR with changes I don't support myself.

**Comment by imagico** [comment]
> I find the new office symbol to be rather cluttered with three different objects. And a person combined with a counter is rather ambiguous, so the 'i' in the background, at least for me, becomes the main recognizable feature.

That is completely fine - i am not tied to the symbol in any way.  It was just a demonstration how you can design a symbol to communicate a more specific meaning (in this case a staffed service place providing information).

> But apart from that I think that adding visitor_centre and changing the office icon are two different things.

As discussed already - just adding `information=visitor_centre` as a synonym for `information=office` is probably not something we want to do.  If the tag is used consistently with a distinctly different meaning than `information=office` then rendering would be viable - but the symbol design for *both* `information=visitor_centre` and `information=office` would need to reflect that.  That is why i am asking what the choice of symbols is meant to communicate.

Note if both tags are consistently used for something specific and distinctly different, using the same symbol for both is in principle an option - we do this elsewhere as well, like for various types of shops.  What we, however, do not want to do is

* rendering both as synonyms without there being a well defined and consistently followed difference in meaning.
* rendering both with different symbols implying there is a semantic difference, but without the symbols used communicating that difference or the difference implied not being followed by mappers in their use of the tags.

**Comment by map-per** [comment]
I updated the test rendering. 
As already mentioned by imagico reviews and comments from others would be nice.

**Comment by danieldegroot2** [comment]
imagico's '"i" icon with roof and roof supports' (is: '"i" within hut symbol')
![image](https://github.com/gravitystorm/openstreetmap-carto/assets/67521919/f6947558-d9c6-4a7a-8104-2479150129da)

The proposed icon looks okay. At least in the original rendering by imagico. It looks a bit small in the PR screenshots.
It at least suggests to some extent it is 'building with information'.

It could suggest [shelter](https://wiki.openstreetmap.org/wiki/File:Shelter-14.svg)/[hut](https://wiki.openstreetmap.org/wiki/File:Alpinehut.svg)/kiosk/telephone box with information, or board/map/terminal/... with roof. It may be too generic to suggest centre, let alone one for visitors of a specific POI (as opposed to information office). However, this could be ignored.

Let's also look at other maps
- Ordinance Survey, United Kingdom (UK) - '"V"' ([source](https://www.ordnancesurvey.co.uk/mapzone/assets/doc/Explorer-25k-Legend-en.pdf)); Seems mostly useful for english-speaking people, ignoring translations.
![image](https://github.com/gravitystorm/openstreetmap-carto/assets/67521919/886a259c-fb5e-4891-a2de-a1f474098c7e)
- Ontario Parks, Canada (CA) - 'roof with under it people reading information board' ([image](https://www.ontarioparks.com/images/svg/icons/visitor_centres.svg), [source](https://www.ontarioparks.com/park/lakesuperior)); More understandable than the first. Seems mostly useful if we don't care if it is staffed and ignore the similarities mentioned above. Might be too close together for a Carto icon.
![image](https://github.com/gravitystorm/openstreetmap-carto/assets/67521919/b2039400-6ec5-4526-8c3e-4b1bd03a5f26) 
- Department of Conservation, New Zealand (NZ) - previously or sometimes used 'building with information' ([source](https://www.flickr.com/photos/alan_cressler/8502813525)); Seems to go for a similar design.
![image](https://github.com/gravitystorm/openstreetmap-carto/assets/67521919/95c063e1-17ce-40f3-ab68-003ad6a2d41e)

---

Sidenotes:

The proposed icon shows some similarity to stock icons for [ranger stations](https://dictionary.cambridge.org/dictionary/english/ranger-station) (namely [variants](https://www.shutterstock.com/nl/image-vector/ranger-station-icon-set-1238743579) of [this icon](https://www.shutterstock.com/nl/image-photo/generic-visitor-center-park-sign-1846709560)); these could also suggest a government building or guard station. A different symbol for ranger stations, ['person giving directions to someone else'](https://www.shutterstock.com/nl/image-photo/lochsa-historic-ranger-station-idaho-usa-532892926), is not much better in this regard.
National Park Service / Department of Agriculture, United States (USA) also sometimes used '"?"' for the visitor centre or a related information POI.
![image](https://github.com/gravitystorm/openstreetmap-carto/assets/67521919/9cb0479e-6e24-4431-9a93-aaff3e4a42d8) / 
![image](https://github.com/gravitystorm/openstreetmap-carto/assets/67521919/df592860-2481-4a70-b7e4-7dc9aef017b6)

When looking from further away (by that point it's actually three stripes with a roof in the PR screenshots), it also shows similarity to [museum](https://wiki.openstreetmap.org/wiki/File:Museum-16.svg), (previously) proposed icons for [social facility](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dsocial_facility#Types_of_social_facilities) types, as well as '"🛐" `U+1F6D0` Place of worship' (compare with [place of worship](https://wiki.openstreetmap.org/wiki/File:Place-of-worship-16.svg))
However, based on the original render by imagico it is unlikely there will be such confusion.

The tagging in general for visitor_centre would probably differentiate actual building you can enter from small building with reception desk inside and boards on the outside. This seems acceptable, though sometimes this is the only thing available on location.

( furthermore, I more or less agree with https://github.com/gravitystorm/openstreetmap-carto/issues/4773#issuecomment-1678728214 )
